src/integrations/clearml/__init__.py

```diff
@@ -0,0 +1,120 @@
+
+__all__ = [
+    'register_to_clearml',
+    'get_clearml_dataset'
+]
+
+import logging
+import re
+from typing import List, Optional
+
+import pandas as pd
+
+from clearml import Dataset
+import tempfile
+
+url_regex = re.compile(
+        r'^(?:http|ftp)s?://'
+        r'(?:(?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\.)+(?:[A-Z]{2,6}\.?|[A-Z0-9-]{2,}\.?)|'
+        r'localhost|'
+        r'\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})'
+        r'(?::\d+)?'
+        r'(?:/?|[/?]\S+)$', re.IGNORECASE)
+
+
+def get_clearml_dataset(
```

Comment:
I believe that this function can be omitted (at least in current form) as it does not provide any value over `Dataset` API. Unlike this function `Dataset.get(...).get_local_copy()` call supports all current and future parameters of the `Dataset.get ` and performs superior validation of the input parameters. Correct me if I am missing some functionality.

Reply:
I believe this function may be useful when one want to label data that is stored in ClearML Dataset

Reply:
> I believe that this function can be omitted (at least in current form) as it does not provide any value over `Dataset` API. Unlike this function `Dataset.get(...).get_local_copy()` call supports all current and future parameters of the `Dataset.get ` and performs superior validation of the input parameters. Correct me if I am missing some functionality.

I believe this function may be useful when one want to label data that is stored in ClearML Dataset